### ISSUE TITLE
Could we have the latest filestack native clients?

### ISSUE THREAD

**Issue by jawgardner** [dbaq/cordova-plugin-filepickerio#7]
Filestack have made some huge improvements under the hood. Would be really great to have those in this lib.

Thanks so much for your hard work and making this open to all of us!

**Comment by dbaq** [comment]
hey @jawgardner, thanks for opening the issue. I have currently no free time to work on an update. We do use this plugin in production as well so I'll have to update this plugin at one point but no short term plan.

**Comment by jawgardner** [comment]
@dbaq ok, thanks for replying. Love the plugin.

**Comment by dbaq** [comment]
hey @jawgardner,

sorry for the delay, can you check out this branch (https://github.com/dbaq/cordova-plugin-filepickerio/pull/14). I updated FSPicker (iOS) to 1.1.7.

The `setName` function is now working for iOS as well.

**Comment by jawgardner** [comment]
Super excited! Thankyou!

**Comment by dbaq** [comment]
hey @jawgardner, have you tested it? I am going to merge it on the master branch but I won't publish a new release on npm until I got some feedbacks.

**Comment by dbaq** [comment]
it is merged @jawgardner! please let me know if something is not working.

also please consider donating to this project if you are using it for a successful app or a company: https://github.com/dbaq/cordova-plugin-filepickerio#donations
thank you

**Comment by jawgardner** [comment]
Apologies for the delay in getting back @dbaq, I've just tested it and it works. Fantastic! I had to manually add the embedded binaries in XCODe, which I am sure will trip many people up, but otherwise, this just works. Love it. Donating now :-)

**Comment by dbaq** [comment]
Thanks for the donation @jawgardner! 

> had to manually add the embedded binaries in XCODe, which I am sure will trip many people up, but otherwise, this just works.

Do you recall which one you had to import manually? I am so used to doing that, I forgot I could be tough for people to figure out how to do it.

I'll publish a npm version soon

**Comment by dbaq** [comment]
Supposedly, all the following libs should be included automatically:

https://github.com/dbaq/cordova-plugin-filepickerio/blob/master/plugin.xml#L44-L53

**Comment by jawgardner** [comment]
I had to manually add:

FSPicker.framework
Filestack.framework
AFNetworking.framework

I am not certain why it didn't happen automatically, but was super grateful for your note at the end of the readme in the FAQ :-)

**Comment by dbaq** [comment]
Glad that helped. I just published version 0.0.8, so we should be all good. Thank you.

```
[8:24:08] ツ dbaq at dbaqs-laptop:~/playground/cordova-plugin-filepickerio on master $ npm publish
+ cordova-plugin-filepickerio@0.0.8
```
